### DISCUSSION TITLE
Pit Level 2 Clear

### DISCUSSION THREAD

**Discussion by eewil** [blizzhackers/kolbot#305] [Ideas]
Hammerdin will take long time clearing Pit level 2 because of Shamans reviving their minions. It would be a good idea to add option killing shamans first then the other monsters. Found a script called AZPit in github but it causes error with the new kolbot branch.

**Comment by mf022** [comment]
maybe "Rest in Peace" ring will resolve this issue, like on the case of Baal wave 2 minions, ... or to use "Redemption" skill from time to time.
